### DISCUSSION TITLE
Ordering Starred Topics

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23982] [New to GitHub]
<p>It would be nice to know if there is a way to choose the order of starred topics without having to star them in a sequence. Is there? (Or is there one coming soon?)</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>There is not a method by which you can order the Topics sidebar at this time.<br>
I can see how that list can get a little unwieldy over time if it gets too large. Being able to customize it, would be quite helpful in those instances.</p>
<p>This does not appear to be in the <a href="https://github.com/github/roadmap">GitHub public roadmap</a>, but you can keep an eye on it for upcoming product changes.</p>
